GroceryStore/InventoryViewController.m

```diff
@@ -0,0 +1,94 @@
+//
+//  InventoryViewController.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "InventoryViewController.h"
+#import "Product.h"
+#import "ProductTableViewCell.h"
+#import "ProductTableView.h"
+
+#import "ProductDetailViewController.h"
+
+@interface InventoryViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (strong, nonatomic) NSArray *productArray;
+@property (weak, nonatomic) IBOutlet ProductTableView *productTableView;
+
+
+@end
+
+@implementation InventoryViewController
+
+- (void)viewWillAppear:(BOOL)animated {
+    
+    [self populateProductArray];
+}
+
+- (ProductTableViewCell *)tableView:(ProductTableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    ProductTableViewCell *productCell = [tableView dequeueReusableCellWithIdentifier:@"productCell" forIndexPath:indexPath];
+    
+    Product *product = [self.productArray objectAtIndex:indexPath.row];
+    
+    productCell.productTitleLabel.text = product.productName;
+    productCell.productQuantityLabel.text = product.quantity;
+    
+    return productCell;
+}
+
+- (NSInteger)tableView:(ProductTableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.productArray.count;
+}
+
+- (void)populateProductArray {
+    self.productArray = [NSArray new];
+    
+    //create session
+    NSString *inventoryURL = @"http://127.0.0.1:5000/api/inventory";
```

Comment:
Ideally, this would be split into host ("http://127.0.0.1:500") and path ("/api/inventory") to avoid duplication and to make it easier to switch hosts (e.g.: dev to staging to prod)